src/averageAges.js

```diff
@@ -13,13 +13,26 @@
  * @param {number} century - optional
  *
  * @return {number}
+ *
  */
+function averageAge(objectWithPeoples) {
+  const allAges = objectWithPeoples.map(human => human.died - human.born);
+
+  return allAges.reduce((sumOFAge, nextAge) =>
+    sumOFAge + nextAge, 0) / allAges.length;
```

Comment:
Не розривай колбек на новий рядок
```suggestion
  return allAges.reduce(
    (sumOFAge, nextAge) => sumOFAge + nextAge,
  0) / allAges.length;
```